docs/releases/2.2.txt

```diff
@@ -260,6 +260,14 @@ Tests
 * The new :setting:`ORACLE_MANAGED_FILES <TEST_ORACLE_MANAGED_FILES>` test
   database setting allows using Oracle Managed Files (OMF) tablespaces.
 
+* The ``SimpleTestCase.allow_database_queries`` and
```

Comment:
The deprecation note goes under "Features deprecated ..." and add a note in docs/internals/deprecation.txt.

Reply:
Will do.